packages/components/pie-chip/src/index.ts

```diff
@@ -97,22 +98,26 @@ export class PieChip extends LitElement implements ChipProps {
             isDismissible,
         } = this;
 
+        const classes = {
```

Comment:
Do any tests need updating?

Reply:
good point! let me take a look 👍

Reply:
@kevinrodrigues no tests check for attribute renders so I am happy - all existing tests passed 😃